cmd/auth.go

```diff
@@ -34,6 +35,8 @@ func AuthCmd(fsets ...*flag.FlagSet) *cobra.Command {
 				return err
 			}
 
+			ttl, _ := cmd.Flags().GetDuration("ttl")
```

Comment:
does this add `--ttl` to the flagset for the auth cmd? also if no `--ttl` is passed then the default for an unchanged/nil Duration == 0? Can we make explicit pls

Reply:
I'm not `cobra` expert. If you know this part better, please make a push.